lib/autoproj/ops/snapshot.rb

```diff
@@ -140,6 +140,8 @@ def snapshot_packages(packages, target_dir = nil, only_local: true)
                         next
                     end
 
+                vcs_info['fingerprint'] = package.autobuild.fingerprint
```

Comment:
This is weird. If the importer supports fingerprinting this should be returned by `#snapshot` from the importer subclass...

Reply:
In this case, `#fingerprint` is calculated by the package (because calculated recursively), not the importer.

In any case, we're discussing moving this to the install importer. We'll do some performance measurement first, just to be sure it's not too expensive.